### PR TITLE
Add business process in merge process

### PR DESCRIPTION
**Pull request by FranckLecuyer**
Branch 'add_business_process_in_merge_process' is built on branch 'mergingWorkflowGui'

### CODE REVIEW

**Review by ChamseddineBhd — APPROVED** [review]

File: src/components/workflows-configuration.js [hunk 1]
```diff
@@ -302,6 +317,18 @@ const WorkflowsContainer = ({
                                     handleAreaNameChanged(e, index)
                                 }
                             />
+                            <TextField
```
Comment (ChamseddineBhd) [inline src/components/workflows-configuration.js]: add  fullWidth={true} please

File: src/utils/api.js [hunk 2]
```diff
@@ -46,7 +46,9 @@ export function connectNotificationsWebsocket(process) {
         wsbase +
         PREFIX_NOTIFICATION_WS +
         '/notify?process=' +
-        encodeURIComponent(process);
+        encodeURIComponent(process) +
+        '?businessProcess=' +
```
Comment (ChamseddineBhd) [inline src/utils/api.js]: & instead of ?

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/components/workflows-configuration.js (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/workflows-configuration.js]: Done

File: src/utils/api.js (hunk 2 above)
Comment (FranckLecuyer) [reply src/utils/api.js]: Corrected

File: src/translations/fr.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "alternativeSourcingActorCode": "Code GRT alternatif (optionnel)",
     "timelineDiagonalLabels": "Afficher les labels en diagonal",
     "name": "nom",
+    "businessProcess": "business process",
```
Comment (ChamseddineBhd) [inline src/translations/fr.json]: french translation !

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/translations/fr.json (hunk 3 above)
Comment (FranckLecuyer) [reply src/translations/fr.json]: Done

**Review by ChamseddineBhd — APPROVED** [review]

**Pull request merged**